typescript/web/src/components/export-button/export-modal/export-card.tsx

```diff
@@ -92,7 +94,17 @@ export const ExportCard = ({
           spacing={4}
           boxSizing="border-box"
         >
-          <Image src={logoSrc} w="16" flexGrow={0} flexShrink={0} />
+          <Image
+            src={logoSrc}
+            w="16"
+            flexGrow={0}
+            flexShrink={0}
+            filter={
+              logoUsesColorMode
+                ? mode("brightness(0)", "brightness(0) invert()")
```

Comment:
When importing the SVG directly from the react files instead of relying on `/static`, you should be able to change colors of the logo without having to hack brightness